src/main/java/uk/gov/companieshouse/api/testdata/service/impl/CompanySearchServiceImpl.java

```diff
@@ -10,20 +10,22 @@
 import uk.gov.companieshouse.api.testdata.model.rest.CompanyData;
 import uk.gov.companieshouse.api.testdata.service.CompanySearchService;
 import uk.gov.companieshouse.logging.Logger;
-import uk.gov.companieshouse.logging.LoggerFactory;
 
 @Service
 public class CompanySearchServiceImpl implements CompanySearchService {
 
     private static final String COMPANY_SEARCH_URI = "/company-search/companies/%s";
     private static final String COMPANY_PROFILE_URI = "/company/%s";
-    private final Supplier<InternalApiClient> internalApiClientSupplier;
+    private static final String ERROR_MSG_ADD_COMPANY = "Failed to insert company profile: ";
+    private static final String ERROR_MSG_DELETE_COMPANY = "Failed to remove company profile: ";
```

Comment:
Could these make reference to search. At the moment it would be hard to work out whether company profile or company profile search succeeded/errored